### ISSUE TITLE
carpetplot and aroma

### ISSUE THREAD

**Issue by cmpetty** [poldracklab/fmriprep#1739]
Is it expected behavior to NOT get the carpetplots for each run when you enable ICA-Aroma?

I like the plots and all the relevant info is in the *desc-confounds_regressors.tsv, just noticed that i no longer received them when enabling aroma.

Thanks

**Comment by effigies** [comment]
I'm pretty sure you're supposed to. Can you share your whole command?

**Comment by cmpetty** [comment]
```
singularity run --cleanenv /usr/local/packages/singularity/images/fmriprep-v1.4.1_biac.simg /mnt/munin3/Gaffrey/NTREC.01/Data/ds001 /mnt/munin3/Gaffrey/NTREC.01/Analysis/output participant --participant-label ${SUBJ} \
 --nthreads 16 --omp-nthreads 16 --skip_bids_validation \
 --output-spaces MNI152NLin6Asym:res-2 \
 --work-dir /mnt/munin3/Gaffrey/NTREC.01/Analysis/output/work \
 --use-aroma --aroma-melodic-dimensionality 30
```



the only difference in my image vs the stock distribution is a tweak in freesurfer to deal with our funky directory permissions that our NFS server is presenting.

**Comment by cmpetty** [comment]
without aroma, i get them.   some subjects still had them, but it was a biproduct of running once before trying aroma.  the plot images have an older timestamp.

**Comment by effigies** [comment]
Right. So it's actually this part `--output-spaces MNI152NLin6Asym:res-2`. We only have the voxel ordering to make a carpet plot for `MNI152NLin2009cAsym` templates, right now.

I would need to dig into it to figure out what's needed to add another template.

**Comment by cmpetty** [comment]
gotcha. thanks for the update.

i actually explicitly set that output-space of aroma because i was trying to avoid having a bunch of different functionals that i wasn't going to use.

**Comment by effigies** [comment]
Yeah, in addition to making it so that functionality is independent of estimated spaces as much as possible, in cases where it's not, we need to dissociate the estimated spaces from the output spaces.

**Comment by oesteban** [comment]
This should have been addressed by #1955 and will be rolled out with version 20.0.0rc1 - please reopen if it regresses.
